<launch>

	<arg name="map_file" default="contest_maze_2017.txt" />  //"contest_maze_2017.txt"

	<node pkg="robot_map" type="map_node" name="map_node" output="screen" >
		 <param name="map_file" value="$(find ras_maze_map)/maps/$(arg map_file)"/>
	</node>

	<node pkg="robot_map" type="map_smoother" name="map_smoother" output="screen" >
		 <param name="map_file" value="$(find ras_maze_map)/maps/$(arg map_file)"/>
	</node>

<remap from="grid_map" to="/map" />


</launch>
